src/test/java/org/opensearch/sdk/TestSDKClusterService.java

```diff
@@ -106,11 +106,9 @@ public void testAddSettingsUpdateConsumer() throws Exception {
     public void testAddSettingsUpdateConsumerMap() throws Exception {
         Setting<Boolean> boolSetting = Setting.boolSetting("test", false);
         Consumer<Boolean> boolConsumer = b -> {};
-        Map<Setting<?>, Consumer<?>> settingsUpdateConsumersMap = new HashMap<>();
```

Comment:
While this works, it ends up duplicating the previous `testAddSettingsUpdateConsumer()` test method.  So this whole test method can be removed.

Reply:
Ok, will update this and make a new PR to the [feature/extension](https://github.com/opensearch-project/anomaly-detection/tree/feature/extensions)